presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestOrderByQueries.java

```diff
@@ -25,7 +25,7 @@
 import static com.facebook.presto.tests.QueryTemplate.parameter;
 import static com.facebook.presto.tests.QueryTemplate.queryTemplate;
 
-public class AbstractTestOrderByQueries
+public abstract class AbstractTestOrderByQueries
```

Comment:
nit: irrelevant change?

Reply:
This is in a separate commit.  I encountered it trying to test the error handling change because it made it annoying to run individual tests through intellij when the class wasn't declared abstract.